AppTests/App/Dependencies/ImmuniExposureDetectionExecutorTests.swift

```diff
@@ -560,15 +667,24 @@ class NoMatchMockExposureNotificationProvider: MockExposureNotificationProvider
 }
 
 class MatchingMockExposureNotificationProvider: MockExposureNotificationProvider {
+  let maximumRiskScore: UInt8
+  let durationByAttenuationBucket: [Double]
+
+  init(overriddenStatus: ExposureNotificationStatus, maximumRiskScore: UInt8 = 8, durationByAttenuationBucket: [Double] = [300, 300, 300]) {
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: AppTests/App/Dependencies/ImmuniExposureDetectionExecutorTests.swift;
  Line: 673;
-->

- :warning: swiftlint (0.39.2): Line should be 130 characters or less: currently 140 characters (line_length)